### PR TITLE
Add Labelled Images and more information about the User Interface

### PR DESCRIPTION
**Pull request by prit3010**
Added Labelled image and more details on the user interface

### CODE REVIEW

**Comment by prit3010** [comment]
Fixes #155

**Review by chao890 — APPROVED** [review]
approved

File: docs/UserGuide.md [hunk 1]
```diff
@@ -86,7 +82,47 @@ Coydir is a desktop app to manage the employee details within a company, optimiz
...
+
+#### Employee Info Panel
+![](images/ui-screenshots/Employee Info Panel.png)
+This shows a specific employee's particulars.
+
+Things to note:
+
+1. On startup of the application, this panel shows Coydir logo.
+2. To view an employee's particulars, refer to the [view command]([#view-details-of-an-employee--view).
+3. If you delete an employee that you are currently viewing, the next employee's details will be viewed.
```
Comment (Kok-je) [inline docs/UserGuide.md:116]: Phrasing: the person info view will display the next employee's deatils

File: docs/UserGuide.md [hunk 2]
```diff
@@ -86,7 +82,47 @@ Coydir is a desktop app to manage the employee details within a company, optimiz
...
+4. If there is no next employee, this panel will revert to the Coydir logo.
+
+
+#### Command Box
+![](images/ui-screenshots/Command.png)
+
+You can enter your user commands in the bottom section labeled as `Enter command here`.
```
Comment (Kok-je) [inline docs/UserGuide.md:123]: Should add the three dots after the line to stay as accurate as possible to the actual application: `Enter command here...`

**Review by Kok-je — CHANGES_REQUESTED** [review]
Overall great work done. Minor edits that are easy to fix.

**Review by Kok-je — APPROVED** [review]
LGLM

**Pull request merged**